spec/controllers/api/v1x0/mixins/destroy_mixin_spec.rb

```diff
@@ -23,6 +24,7 @@
       let!(:endpoint_1) { create(:endpoint, :role => "a", :source => source_1) }
 
       it "delete /sources/:id/endpoint/:id fails with 404" do
+        stub_const("ENV", "BYPASS_RBAC" => "false")
```

Comment:
I think most of these can be removed - we _should_ only need the stub at the top of each file, so only once per file. I tested a few files locally and with the top before set it should run the stub before each example in the group. Sorry to be such a pain but it should be as easy as pulling all the stubs out minus the one at the top of every file (for most of them - there might be a few that need it).